packages/suite-desktop-core/e2e/support/pageObjects/stakingSection.ts

```diff
@@ -20,6 +20,7 @@ export class StakingSection {
     readonly unstakeButton: Locator;
     readonly speedUpButton: Locator;
     readonly pendingTransactionText: Locator;
+    readonly unconfirmedTransactionText: Locator;
```

Comment:
Rather then adding another locator, we need to replace the original. And replace all its uses.
In tests we first check for visibility of the now unconfirmed transaction and later that is gone. 
I have noticed that you did update only the first part, that it is visible and the second on is falsely passing. Because it is rellying on the old locator which never was there in first place.
```suggestion
    readonly unconfirmedTransactionText: Locator;
````